pytest_postgresql/executor.py

```
@@ -145,6 +177,32 @@ def start(self: T) -> T:
+            args = [
+                self.executable,
+                "start",
+                "-D",
+                self.datadir,
+                "-o",
+                pg_options,
+                "-l",
+                self.logfile,
+                *shlex.split(self.startparams),
+            ]
+            result = subprocess.run(args, check=False, env=self.envvars)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)